hieradata/role/fiberspec.yaml

```diff
@@ -4,4 +4,9 @@ classes:
   - "profile::core::docker"
   - "profile::core::fiberspec"
   - "profile::core::nfsclient"
+  - "profile::core::systemd"
   - "profile::ts::dco"
+
+systemd::manage_udevd: true
+systemd::udev_log: 'debug'
+systemd::udev_reload: true
```

Comment:
Should we set this in the `common` layer? I think it is probably the behavior we want for all nodes.

Reply:
I think for now its a good idea.